doc/manpages/vtysh.rst

```diff
@@ -67,6 +67,15 @@ OPTIONS available for the vtysh command:
 
    Display a usage message on standard output and exit.
 
+.. option:: -t, --timestamp
```

Comment:
I'd like to conserve short-option characters, mostly to have the same character with the same meaning across as much as FRR as possible — can we ditch `-s` and `-t` here and have long options only for these two?

(`-s` is `--nl-bufsize` in zebra; `-t` is `--terminal` in all daemons)

Reply:
`-s` sure, talk to @choppsv1 about `-t`

Reply:
I'm fine with dumping the `-t` might have to look for and fix uses in the tree.